# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/child/FileSystemBackgroundRequestHandler.h
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/child/FileSystemBackgroundRequestHandler.h
--- a/dom/fs/child/FileSystemBackgroundRequestHandler.h	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/child/FileSystemBackgroundRequestHandler.h	Wed Aug 31 09:34:48 2022 +0000
@@ -21,7 +21,7 @@
 
 namespace dom {
 
-class OriginPrivateFileSystemChild;
+class FileSystemManagerChild;
 
 namespace fs {
 class FileSystemChildFactory;
@@ -35,7 +35,7 @@
   FileSystemBackgroundRequestHandler();
 
   using CreateFileSystemManagerChildPromise =
-      MozPromise<RefPtr<OriginPrivateFileSystemChild>, nsresult, false>;
+      MozPromise<RefPtr<FileSystemManagerChild>, nsresult, false>;
 
   virtual RefPtr<CreateFileSystemManagerChildPromise>
   CreateFileSystemManagerChild(